<?xml version="1.0"?>
<launch>

<arg name="tag_param" default="[{id: $(env DOCK_ID),size: 0.100}]"/>

<node pkg="apriltags_ros" type="apriltag_detector_node" name="apriltag_detector">
    <rosparam param="tag_descriptions" subst_value="True" >$(arg tag_param)</rosparam>
    <remap from="/image_rect" to="/camera/rgb/image_raw"/>
    <remap from="/camera_info" to="/camera/rgb/camera_info"/>
    <remap from="/tag_detections" to="docking_tags"/> 
 </node>


<!-- Listener -->
<node name="dock" pkg="docking" type="dock" args="--test" respawn="false" output="screen" >
<param name="tag_id" type="str" value="$(env DOCK_ID)" />

</node>
</launch>

